types/electra/attestation.yaml

```diff
@@ -0,0 +1,33 @@
+Electra:
+  IndexedAttestation:
+    type: object
+    description: "The [`IndexedAttestation`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.2/specs/electra/beacon-chain.md#indexedattestation) object from the CL spec."
+    properties:
```

Comment:
`required` properties filed is missing here

Reply:
Good catch, they were missing in the existing `IndexedAttestaion`, added them there too